src/NServiceBus.Core/Routing/RoutingSettingsExtensions.cs

```diff
@@ -10,10 +10,10 @@ public static class RoutingSettingsExtensions
         /// <summary>
         /// Controls the unicast routing.
         /// </summary>
-        public static UnicastRoutingSettings UnicastRouting(this ExposeSettings config)
+        public static RoutingSettings Routing(this ExposeSettings config)
```

Comment:
I'd really like to see this changed to `EndpointConfiguration` instead of `ExposeSettings` to get rid of the `Routing` extension showing up where it doesn't belong, for example being able to do this:

`config.UnicastRouting().UnicastRouting();`

or
`config.UnicastRouting().Mapping.UnicastRouting();`